Elevate.podspec

```diff
@@ -1,13 +1,14 @@
 Pod::Spec.new do |s|
   s.name = 'Elevate'
-  s.version = '3.0.0'
+  s.version = '3.0.1'
   s.summary = 'Elevate is a JSON parsing framework that leverages Swift to make parsing simple, reliable and composable.'
   s.homepage = 'https://github.com/Nike-Inc/Elevate'
   s.license = 'MIT'
   s.authors = { 'Eric Appel' => 'eric.appel@nike.com', 'Christian Noon' => 'christian.noon@nike.com' }
 
   s.source = { :git => 'https://github.com/Nike-Inc/Elevate.git', :tag => s.version }
   s.source_files = 'Source/*.swift'
+  s.swift_version = '4.0'
```

Comment:
4.1?

Reply:
Should be `4.0` compatible. The `NSNumber` changes just make the code do what it should have done previously.

Reply:
Yep, just verified all these changes work properly on Xcode 9.2.